res/css/views/elements/_SSOButtons.scss

```diff
@@ -46,13 +46,13 @@ limitations under the License.
     }
 
     .mx_SSOButton_default {
-        color: $button-primary-bg-color;
-        background-color: $button-secondary-bg-color;
-        border-color: $button-primary-bg-color;
+        color: $button-secondary-bg-color;
```

Comment:
These changes look odd, as the default button is being assigned the secondary colours which is not the expected behaviour.

Reply:
this change needed for the light color button now instead of the white button previously for SSO

Reply:
done